js/__internal/grids/grid_core/keyboard_navigation/module.ts

```diff
@@ -1,115 +1,121 @@
-// @ts-check
+import { focused } from '@js/ui/widget/selectors';
+import { noop } from '@js/core//utils/common';
 
+/* eslint-disable class-methods-use-this */
 import {
-  getOuterHeight, getHeight, getWidth, getOuterWidth,
+  getOuterHeight,
+  getHeight,
+  getWidth,
+  getOuterWidth,
 } from '@js/core/utils/size';
-import $ from '@js/core/../core/renderer';
-import domAdapter from '@js/core/../core/dom_adapter';
-import eventsEngine from '@js/core/../events/core/events_engine';
-import { isDefined, isEmptyObject } from '@js/core/../core/utils/type';
-import { focused } from '@js/ui/widget/selectors';
-import { addNamespace, createEvent, isCommandKeyPressed } from '@js/core/../events/utils/index';
+import $ from '@js/core/renderer';
+import domAdapter from '@js/core/dom_adapter';
+import eventsEngine from '@js/events/core/events_engine';
+import { isDefined, isEmptyObject } from '@js/core/utils/type';
+import {
+  addNamespace,
+  createEvent,
+  isCommandKeyPressed,
+} from '@js/core/../events/utils/index';
```

Comment:
Let's remove this double dots:

```suggestion
} from '@js/events/utils/index';
```